ArduCopter/mode_stabilize.cpp

```diff
@@ -62,6 +62,9 @@ void ModeStabilize::run()
         break;
     }
 
+    // reduce gains when landed;
+    attitude_control->landed_gain_reduction(copter.ap.land_complete);
```

Comment:
Only stabilize?

Reply:
@lthall do you intend to have this work in all modes at some point. I'm curious like @andyp1per as to why this is just stabilize

Reply:
Yes, this is just an example of how I need to do it right now. I was looking for suggestions on how I could do it better.